packages/create-hops-app/package.json

```diff
@@ -11,7 +11,8 @@
     "node": "^10 || ^12"
   },
   "bin": {
-    "create-hops-app": "./index.js"
+    "create-hops-app": "./index.js",
+    "create-hops-app@next": "./index.js"
```

Comment:
One more thing: Last time we forgot to remove `create-hops-app@next` from the bin section after releasing the stable release.
Could we add another test, that reads the bin entries and verifies, that they are correct?

Reply:
I guess there won't be a clean transition path from pre-release to release, but at least we're aware of the issue. 🤷‍♂️